Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -226,6 +226,28 @@ public enum GeneralTraitsOptions {
   public static readonly Option<bool> IncludeRuntimeOption = new("--include-runtime",
     "Include the Dafny runtime as source in the target language.");
 
+  /// <summary>
+  /// Copy of --include-runtime for execution commands like `dafny run`,
+  /// just so it can be internal only in that context:
+  /// it shouldn't matter to end users but is useful for testing.
+  /// </summary>
+  public static readonly Option<bool> InternalIncludeRuntimeOptionForExecution = new("--include-runtime", () => true,
+    "Include the Dafny runtime as source in the target language.") {
+    IsHidden = true
+  };
+
+  public enum SystemModuleMode {
+    Include,
+    Omit,
+    // TODO: better name? OmitOthers?
```

Comment:
The rule of these names seems to be to describe the effect, not the reason behind it, so `OmitAllOtherModules` or `OnlyIncludeSystem[Module]` matches well.